server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/MetricKeys.java

```diff
@@ -85,4 +85,8 @@
     public static final String REPLICATION_COUNT = "replication-count";
     public static final String REPLICATION_FAILURES = "replication-failures";
     public static final String SUCCESS_RATIO = "success-ratio";
+    // Backup Metrics
+    public static final String ONLINE_SITES = "online-sites";
+    public static final String OFFLINE_SITES = "offline-sites";
+    public static final String MIXED_SITES = "mixed-sites";
```

Comment:
Ok, let's reverse the strings then. "sites-online", "sites-offline", "sites-mixed"